env.default

```diff
@@ -1,6 +1,41 @@
+# Build system environment:
+
 PULSE_API_DOMAIN=network-pulse-api-production.herokuapp.com
 PULSE_DOMAIN=www.mozillapulse.org
-NETWORK_API_DOMAIN=network.mofoprod.net
+NETWORK_API_DOMAIN=network.mozilla.org
 TARGET_DOMAIN=network.mofostaging.net
 JENKINS_API=https://jenkins.mofoprod.net/job/Network%20(production)/lastSuccessfulBuild/api/json
 SHOW_TAKEOVER=false
+
+# network-api environment:
+
+DEBUG=True
+DJANGO_SECRET_KEY=secret
+USE_S3=False
+CORS_WHITELIST=*
+CORS_REGEX_WHITELIST=
+XSS_PROTECTION=True
+CONTENT_TYPE_NO_SNIFF=True
+SET_HSTS=False
+SSL_REDIRECT=False
+X_FRAME_OPTIONS=DENY
+ALLOWED_HOSTS=localhost
+ASSET_DOMAIN=network.mofoprod.net
+BUILD_TRIGGER_URL=
+BUILD_DEBOUNCE_SECONDS=300
+BUILD_THROTTLE_SECONDS=900
+SOCIAL_AUTH_GOOGLE_OAUTH2_KEY=
+SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET=
+SOCIAL_AUTH_LOGIN_REDIRECT_URL=http://localhost:5000/soc/complete/google-oauth2/
+
+# CSP config
```

Comment:
does our env loader understand comments, since this isn't a bash-style `source` call?

Reply:
[It reportedly does](https://github.com/joke2k/django-environ)

Reply:
It seems to. On the build system side it converts to JSON and just ignores comments as expected.